tools/ci/tcdownload.py

```diff
@@ -74,8 +76,13 @@ def run(*args, **kwargs):
         return 1
 
     for taskgroup in taskgroups:
-        taskgroup_url = "https://queue.taskcluster.net/v1/task-group/%s/list"
-        artifacts_list_url = "https://queue.taskcluster.net/v1/task/%s/artifacts"
+        if TASKCLUSTER_ROOT_URL == 'https://taskcluster.net':
+            # NOTE: this condition can be removed after November 9, 2019
```

Comment:
It would be helpful for this comment to contain the new url like the one in `browser.py`

Reply:
The whole branch of the if can be removed, leaving just the content of the `else` branch, which depends on the value of TASKCLUSTER_ROOT_URL.

Reply:
Right, I just meant it would be useful to document that you also need to switch `TASKCLUSTER_ROOT_URL` to (whatever the new value is which I'll need to look up) when you delete the if condition.

Reply:
Ah, got it.